src/mono/mono/mini/mini-runtime.c

```diff
@@ -4565,6 +4589,11 @@ mini_init (const char *filename)
 	mono_w32handle_init ();
 #endif
 
+#ifdef ENSURE_PRIMARY_STACK_SIZE 
+	// TODO: https://github.com/dotnet/runtimelab/issues/791
```

Comment:
looks like this issue was moved:

```suggestion
	// TODO: https://github.com/dotnet/runtime/issues/72920
```